app/services/fastboot.js

```diff
@@ -75,12 +73,6 @@ const FastBootService = Ember.Service.extend({
   },
 
   host: computed(function() {
```

Comment:
No the `host` property is public API: http://ember-fastboot.com/docs/user-guide#request-host